packages/components/src/types/page.ts

```diff
@@ -87,11 +98,15 @@ export const CollectionPagePageSchema = Type.Intersect([
     }),
   }),
   TagsSchema,
+  fallbackIfNoImageSchemaObject,
 ])
 
-export const ContentPagePageSchema = Type.Object({
-  contentPageHeader: ContentPageHeaderSchema,
-})
+export const ContentPagePageSchema = Type.Intersect([
+  Type.Object({
+    contentPageHeader: ContentPageHeaderSchema,
+  }),
+  fallbackIfNoImageSchemaObject,
+])
```

Comment:
question: hmm curious why did we add this to the `content` layout? Based on the rest of the code it seems to be a toggle that is used more on the `collection` layout.